dts/bindings/dma/st,stm32-dma-v2.yaml

```diff
@@ -17,35 +17,62 @@ description: |
     this value is 0 for Memory-to-memory transfers
     or a value between <1> .. <dma-generators> (not supported yet)
     or a value beweeen <dma-generators>+1  ..  <dma-generators>+<dma-requests>
-    3. channel-config: A 32bit mask specifying the DMA channel configuration
-    which is device dependent:
-        -bit 6-7 : Direction  (see dma.h)
-               0x0: MEM to MEM
-               0x1: MEM to PERIPH
-               0x2: PERIPH to MEM
-               0x3: reserved for PERIPH to PERIPH
-        -bit 9 : Peripheral Increment Address
-               0x0: no address increment between transfers
-               0x1: increment address between transfers
-        -bit 10 : Memory Increment Address
-               0x0: no address increment between transfers
-               0x1: increment address between transfers
-        -bit 11-12 : Peripheral data size
-               0x0: Byte (8 bits)
-               0x1: Half-word (16 bits)
-               0x2: Word (32 bits)
-               0x3: reserved
-        -bit 13-14 : Memory data size
-               0x0: Byte (8 bits)
-               0x1: Half-word (16 bits)
-               0x2: Word (32 bits)
-               0x3: reserved
-        -bit 15: Reserved
-        -bit 16-17 : Priority level
-               0x0: low
-               0x1: medium
-               0x2: high
-               0x3: very high
```

Comment:
Please keep this, it is still valuable, clear and readable information.

Reply:
What about:
        -bit 6-7 : Direction  (see dma.h)
               0x0: STM32_DMA_MEMORY_TO_MEMORY: MEM to MEM
               0x1: STM32_DMA_MEMORY_TO_PERIPH: MEM to PERIPH                              
               0x2: STM32_DMA_PERIPH_TO_MEMORY: PERIPH to MEM
               0x3: reserved for PERIPH to PERIPH